rdr_service/code_constants.py

```diff
@@ -110,6 +110,7 @@
 LIFESTYLE_PPI_MODULE = "Lifestyle"
 THE_BASICS_PPI_MODULE = "TheBasics"
 FAMILY_HISTORY_MODULE = "FamilyHistory"
+SOCIAL_DETERMINANTS_OF_HEALTH_MODULE = "sdoh_intro"
```

Comment:
It looks like they must have updated the Redcap project after this was synced. The codebook in Redcap looks like the module code should be `sdoh` instead (RDR would need to be re-synced with Redcap too).

Reply:
Done